lib/rtcicecandidate.js

```diff
@@ -1,12 +1,10 @@
-/* global mozRTCIceCandidate, RTCIceCandidate */
+/* global RTCIceCandidate */
 'use strict';
 
-if (typeof RTCIceCandidate !== 'undefined') {
```

Comment:
we should all this checks for webRTC related types into `isWebRTCSupported` and use it here.

Reply:
@makarandp0 Since this module depends on a specific API, its OK to check on only that API.